openwisp_utils/admin_theme/admin.py

```diff
@@ -18,6 +27,70 @@ class OpenwispAdminSite(admin.AdminSite):
     )
     enable_nav_sidebar = False
 
+    def login(self, *args, **kwargs):
+        response = super().login(*args, **kwargs)
+        if isinstance(response, HttpResponseRedirect):
+            response = HttpResponseRedirect(reverse('admin:ow_dashboard'))
+        return response
+
+    def ow_dashboard(self, request, extra_context=None):
+        context = {
+            'is_popup': False,
+            'has_permission': True,
+        }
+        schema = copy.deepcopy(DASHBOARD_SCHEMA)
+
+        for key, value in schema.items():
+            app_label, model_name, group_by = value['query_params'].values()
+            try:
+                model = load_model(app_label, model_name)
+            except ImproperlyConfigured:
+                raise ImproperlyConfigured(
+                    f'Error adding dashboard element {key}.'
+                    f'REASON: {app_label}.{model_name} could not be loaded.'
+                )
+
+            # Filter query according to organization of user
+            if hasattr(model, 'organization_id') and not request.user.is_superuser:
```

Comment:
I would avoid the if since at the moment we only plan to show elements which are bound to organizations so we should focus on that

Reply:
I think it is better to keep it for future scope. Moreover, I don't think the graph added by the `test_project` will be able to work without this condition.